core/src/main/java/org/frankframework/util/flow/graphviz/GraphvizEngine.java

```diff
@@ -181,14 +181,16 @@ public void close() {
 	}
 
 	private String getVisJsWrapper() {
-		return "var viz = new Viz();"
-				+ "function render(src, options){"
-				+ "  try {"
-				+ "    viz.renderString(src, options)"
-				+ "      .then(function(res) { result(res); })"
-				+ "      .catch(function(err) { viz = new Viz(); error(err.toString()); });"
-				+ "  } catch(e) { error(e.toString()); }"
-				+ "}";
+		return """
+				var viz = new Viz();\
+				function render(src, options){\
+				  try {\
+				    viz.renderString(src, options)\
+				      .then(function(res) { result(res); })\
+				      .catch(function(err) { viz = new Viz(); error(err.toString()); });\
+				  } catch(e) { error(e.toString()); }\
+				}\
```

Comment:
Should we remove those added `\` chars?

Reply:
Lijkt mij wel.

Reply:
Heb het geprobeerd, maar dan falen er unittesten. Blijkbaar betekent wat vroeger een `\n` was, dat het wordt omgezet naar een nieuwe regel met text-blocks. En als er geen nieuwe regel moet komen, maar dat er wel staat, er alleen een `\` wordt neergezet.